src/typing_extensions.py

```diff
@@ -2545,7 +2545,7 @@ def __typing_is_unpacked_typevartuple__(self):
         def __getitem__(self, args):
             if self.__typing_is_unpacked_typevartuple__:
                 return args
-            return super().__getitem__(args)
+            return typing._GenericAlias.__getitem__(self, args)
```

Comment:
```suggestion
            # Cannot use `super().__getitem__` here because of the `__class__` assignment
            # in the class body on Python <=3.11
            # (https://github.com/python/typing_extensions/issues/661)
            return typing._GenericAlias.__getitem__(self, args)
```